client/my-sites/plans/jetpack-plans/faq/index.tsx

```diff
@@ -63,19 +62,29 @@ const JetpackFAQ: React.FC = () => {
 						}
 					) }
 				</FoldableFAQ>
-
+				<FoldableFAQ
+					id="volume-discount"
+					question={ translate( 'Do you have any discounts for multiple sites?' ) }
+					onToggle={ onFaqToggle }
+				>
+					{ translate(
+						'Anyone with at least five websites can join our licensing platform and enjoy a 25% discount across all Jetpack products! You can learn more about our {{agenciesLink}}licensing platform and agency program here{{/agenciesLink}}.',
+						{
+							components: { agenciesLink: getAgenciesLink() },
+						}
+					) }
+				</FoldableFAQ>
 				<FoldableFAQ
 					id="wpcom-account"
 					question={ translate( 'Why do I need a WordPress.com account?' ) }
 					onToggle={ onFaqToggle }
 				>
 					{ translate(
-						"Many of Jetpack's core features make use of the WordPress.com cloud. In order to make sure everything" +
-							" works correctly, Jetpack requires you to connect a free WordPress.com account. If you don't already" +
+						'Many of Jetpack’s core features make use of the WordPress.com cloud. In order to make sure everything' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Many of Jetpack's core features make use of the WordPress.com cloud. In order to make sure everything works correctly, Jetpack requires you to connect a free WordPress.com account. If you don't already have an account you can easily create one during the connection process.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-pricing-page-faq-agencies-discount)